src/libstore/globals.hh

```diff
@@ -11,7 +11,7 @@
 
 namespace nix {
 
-typedef enum { smEnabled, smRelaxed, smDisabled } SandboxMode;
+typedef enum { smEnabled, smEnabledIfAvailable, smRelaxed, smDisabled } SandboxMode;
```

Comment:
Maybe this should be a separate flag, because you may want to use relaxed mode *and* disable the sandbox if unavailable.